rts/Rendering/WorldDrawer.cpp

```diff
@@ -48,6 +49,12 @@
 void CWorldDrawer::InitPre() const
 {
 	CShaderHandler::GetInstance(0);
+
+	loadscreen->SetLoadMessage("Creating Deferred Buffers");
+	GL::GeometryBufferUni::geomBuffer = new GL::GeometryBuffer("UNIFIED - GBUFFER");
+	if (GL::GeometryBufferUni::geomBuffer->EnabledAndValid(true))
```

Comment:
these lines and the `SafeDelete` downstairs can go in their own functions `GL::GeometryBufferUni::Create()` and `GL::GeometryBufferUni::Destroy()`